src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -7,6 +7,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
     <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(TargetOS)' == 'Browser'">true</DebuggerSupport>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <IsHighAotMemoryUsageTest>true</IsHighAotMemoryUsageTest>
```

Comment:
NIT: Can we add a comment above this property to explain what it does and why is it needed? from a quick search seems like this is only relevant for wasm right?

Reply:
> only relevant for wasm right?

I assume so.  I'm just moving it over from the old generators test project.  It was added in https://github.com/dotnet/runtime/commit/5a078b513fd4955306ecd5c9d9418bcf87416d09.